### PR TITLE
DOC-5889 - overview text of db in getting started

### PR DESCRIPTION
**Pull request by jamesnocentini**
https://issues.couchbase.com/browse/DOC-5889

Introduce the concept of a Sync Gateway database and how it relates to a bucket

### CODE REVIEW

File: modules/ROOT/pages/getting-started.adoc [hunk 1]
```diff
@@ -475,7 +475,12 @@ The configuration file and logs are located in `/Users/sync_gateway`.
 
 The following steps explain how to connect Sync Gateway to the Couchbase Server instance.
 
-* Open a new file called *sync-gateway-config.json* with the following.
+* You must first create a Sync Gateway *database*.
+A Sync Gateway database is a namespace for documents and multiple databases can be defined in the configuration file for Sync Gateway.
+Each Sync Gateway *database* connects to a Couchbase Server *bucket*.
```
Comment (rajagp) [inline modules/ROOT/pages/getting-started.adoc:480]: A Sync Gateway can be configured with several databases, each of which points to a Couchbase Server bucket.

**Review by rajagp — COMMENTED** [review]
@jamiltz  I think @adamcfraser may have a better definition for a Sync Gateway database.

File: modules/ROOT/pages/getting-started.adoc (hunk 1 above)
Comment (ibsoln) [reply modules/ROOT/pages/getting-started.adoc:480]: @rajagp does this still need changing?

**Comment by rajagp** [comment]
@adamcfraser  Do you have a good definition of a "Sync Gateway Database" ? The proposal is "A Sync Gateway database is a namespace for documents" and I am not sure if that's the best definition.

**Pull request closed without merging**